pkg/apis/devops/v1alpha2/register.go

```diff
@@ -87,7 +95,8 @@ func addWebService(c *restful.Container) error {
 		Param(webservice.PathParameter("pipelineName", "pipeline name")).
 		Param(webservice.PathParameter("projectName", "devops project name")).
 		Param(webservice.PathParameter("branchName", "pipeline branch name")).
-		Param(webservice.PathParameter("runId", "pipeline runs id")))
+		Param(webservice.PathParameter("runId", "pipeline runs id")).
+		Returns(200, "success", modelDevops.RunPipeline{}))
```

Comment:
use status code `200`, `success` as constant here

Reply:
ok,and I use `ok` for description message, other api use the default message is `ok`.